build.gradle

```diff
@@ -7,16 +7,22 @@ buildscript {
         classpath "org.shipkit:shipkit-changelog:1.2.0"
         classpath "org.shipkit:shipkit-auto-version:1.2.2"
         classpath "io.github.gradle-nexus:publish-plugin:1.3.0"
+        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:1.9.20"
+        classpath "org.jetbrains.dokka:dokka-gradle-plugin:1.9.10"
     }
 }
 
+apply plugin: 'org.gradle.base'
 apply plugin: "io.github.gradle-nexus.publish-plugin"
 apply plugin: 'org.shipkit.shipkit-auto-version'
 apply plugin: "org.shipkit.shipkit-changelog"
 apply plugin: "org.shipkit.shipkit-github-release"
 
-allprojects {
-    group = 'org.mockito.kotlin'
```

Comment:
12. moved this to `:mockito-kotlin/build.gradle` ONLY.

The CI failed with an error:
```
* What went wrong:
Could not determine the dependencies of task ':tests:test'.
> Could not resolve all task dependencies for configuration ':tests:testRuntimeClasspath'.
   > Could not resolve org.mockito.kotlin:mockito-kotlin.
     Required by:
         project :tests
      > Module version 'org.mockito.kotlin:mockito-kotlin' is not unique in composite: can be provided by [project :, project :mockito-kotlin].
```

which puzzled me because locally it was working perfectly.

I understand now what was the problem: I cloned the repo as `github-mockito-kotlin`, but the repo name is `mockito-kotlin` and `settings.gradle` does not contain a `rootProject.name` therefore the default is `mockito-kotlin`.

This above and the `allprojects` means that there are really two `org.mockito.kotlin:mockito-kotlin` artifacts, even though the rootProject (`:`) is not published.

I had the choice of setting some random `mockito-kotlin-repo` or `mockito-kotlin-root` name in `settings.gradle`, but I went with moving the `group`, because that `allprojects` is evil anyway and not necessary here as there are no other modules.